tuf/client/updater.py

```diff
@@ -1151,12 +1151,12 @@ def neither_403_nor_404(mirror_error):
           # Otherwise, reraise the error, because it is not a simple HTTP
           # error.
           if neither_403_nor_404(mirror_error):
-            logging.exception('Misc error for root version '+str(next_version))
+            logger.exception('Misc error for root version '+str(next_version))
```

Comment:
💯 excellent find. I've looked at this code numerous times and not noticed the typo here.

Reply:
My bad, thanks everyone